aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -39,6 +39,19 @@ The Generic Host library is available in the [Microsoft.Extensions.Hosting names
 
 [!code-csharp[](generic-host/samples-snapshot/2.x/GenericHostSample/Program.cs?name=snippet_HostBuilder)]
 
+## Default services
+
+The following services are registered during host initialization:
+
+* [Environment](xref:fundamentals/environments) (<xref:Microsoft.AspNetCore.Hosting.IHostingEnvironment>)
```

Comment:
Microsoft.Extensions.Hosting.IHostingEnvironment

Some of these types were duplicated into the Extensions.Hosting packages/namespaces. There shouldn't be any AspNetCore.Hosting references here.